src/view/com/composer/Composer.tsx

```diff
@@ -1213,17 +1213,30 @@ export function useComposerCancelRef() {
   return useRef<CancelRef>(null)
 }
 
-function useAnimatedBorders() {
+function useScrollTracker({
+  scrollViewRef,
+  stickyBottom,
+}: {
+  scrollViewRef: AnimatedRef<Animated.ScrollView>
+  stickyBottom: boolean
+}) {
   const t = useTheme()
-  const hasScrolledTop = useSharedValue(0)
-  const hasScrolledBottom = useSharedValue(0)
```

Comment:
NB: I flipped their names/meaning because the old names were ambiguous. https://github.com/bluesky-social/social-app/pull/6045/commits/61e928b21394a52ca7b12fd04108f4516c30716f